src/CodeStyle/Directory.Build.targets

```diff
@@ -1,7 +1,7 @@
 <Project>
   <Import Project="..\..\Directory.Build.targets"/>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsShipping)' == 'true'">
```

Comment:
❔ Why was this change needed?

Reply:
This was firing in the some of the shared legacy test framework files - let me see if I can revert it now.

Reply:
Yeah, was able to revert this without problem.